.circleci/config.yml

```diff
@@ -7,23 +7,69 @@ executors:
   node:
     working_directory: /mnt/ramdisk
     docker:
-      - image: node:14
+      - image: node:16
         user: node
   puppeteer:
     working_directory: /mnt/ramdisk
     docker:
-      - image: circleci/node:14-browsers
+      - image: circleci/node:16-browsers
   gcloud:
     working_directory: /mnt/ramdisk
     docker:
       - image: google/cloud-sdk:alpine
 
 commands:
+  notify-github: # borrows from https://circleci.com/developer/orbs/orb/mmnfst/github-deployments#orb-source
+    parameters:
+      env:
+        type: string
+      pr:
+        type: boolean
+      existing:
+        type: boolean
+        default: false
+    steps:
+      - unless:
+          condition: << parameters.existing >>
+          steps:
+            - run:
+                name: Notify Github of deployment
+                unless: << parameters.existing >>
+                command: |
+                  apk add jq -q --no-progress
+                  <<# parameters.pr >>
+                  VERSION="pr-$(expr "${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" % 10)"
+                  <</ parameters.pr >><<^ parameters.pr >>
+                  VERSION="<< parameters.env >>"
+                  <</ parameters.pr >>
+                  curl -d "{ \"ref\": \"$CIRCLE_BRANCH\", \"environment\": \"$VERSION\", \"auto_merge\": false, \"required_contexts\": []}" \
+                    -H "Authorization: token $GITHUB_TOKEN" -XPOST \
+                    https://api.github.com/repos/databiosphere/terra-ui/deployments | jq .id > /tmp/build_num
+      - when:
+          condition: << parameters.existing >>
+          steps:
+            - run:
+                name: Update Github deployment status
+                command: |
+                  <<# parameters.pr >>
+                  DEPLOY_SLUG="pr-$(expr "${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" % 10)-dot-"
+                  <</ parameters.pr >>
+                  curl -d "{ \"state\": \"success\", \"environment_url\": \"https://${DEPLOY_SLUG}bvdp-saturn-<< parameters.env >>.appspot.com\", \"log_url\": \"$CIRCLE_BUILD_URL\" }" \
+                    -H "Authorization: token $GITHUB_TOKEN" -XPOST \
+                    https://api.github.com/repos/databiosphere/terra-ui/deployments/$(cat /tmp/build_num)/statuses
+            - run:
+                name: Update Github deployment status
+                when: on_fail
+                command: |
+                  curl -d "{ \"state\": \"failure\" }" \
+                    -H "Authorization: token $GITHUB_TOKEN" -XPOST \
+                    https://api.github.com/repos/databiosphere/terra-ui/deployments/$(cat /tmp/build_num)/statuses
+
   deploy-env:
     description: "Deploy to a specific environment"
     parameters:
       sa_key_var:
-        type: string
+        type: env_var_name
```

Comment:
`env_var_name` is a valid `type`?

Reply:
nevermind - found the [doc](https://circleci.com/docs/2.0/reusing-config/#environment-variable-name) 😊